frame/support/src/storage/bounded_vec.rs

```diff
@@ -188,11 +201,26 @@ impl<T: BoundedVecValue, S: Get<u32>> sp_std::ops::Deref for BoundedVec<T, S> {
 	}
 }
 
+impl<T: BoundedVecValue, S: Get<u32>> sp_std::ops::DerefMut for BoundedVec<T, S> {
```

Comment:
This isn't a good idea, as pointed out [here](https://github.com/paritytech/substrate/pull/8707#pullrequestreview-649727633).